### PR TITLE
Update pixel updates to 7 years

### PR DESCRIPTION
**Pull request by efb4f5ff-1298-471a-8973-3d47447115dc**
Changes proposed in this PR:

- Now that Google Pixel 8 is released it also comes with 7 years of OS upgrades and 7 years of security updates

Also makes price easier to justify:
https://github.com/privacyguides/privacyguides.org/blob/387f191ba99f7f3e0afd6906aa8b01811d9c9443/docs/android.md?plain=1#L193

- [x] I have disclosed any relevant conflicts of interest in my post.
- [x] I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
- [x] I am the sole author of this work. 
- [x] I agree to the [Community Code of Conduct](https://www.privacyguides.org/en/code_of_conduct/).

### CODE REVIEW

File: docs/android.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ Google Pixel phones are the **only** devices we recommend for purchase. Pixel ph
 
     **Google Pixel** devices are known to have good security and properly support [Verified Boot](https://source.android.com/security/verifiedboot), even when installing custom operating systems.
 
-    Beginning with the **Pixel 6** and **6 Pro**, Pixel devices receive a minimum of 5 years of guaranteed security updates, ensuring a much longer lifespan compared to the 2-4 years competing OEMs typically offer.
+    Beginning with the **Pixel 8** and **8 Pro**, Pixel devices receive a minimum of 7 years of guaranteed security updates, ensuring a much longer lifespan compared to the 2-4 years competing OEMs typically offer.
```
Comment (jonaharagon) [inline docs/android.md:178]:
```suggestion
    Beginning with the **Pixel 8** and **8 Pro**, Pixel devices receive a minimum of 7 years of guaranteed security updates, ensuring a much longer lifespan compared to the 4-5 years competing OEMs typically offer.
```

As far as I know Samsung does 4 minimum, OnePlus does 4 minimum, Fairphone does 5 minimum.

Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply docs/android.md:178]: Aren't there any big phone manufactures out there that still do 2 years?

Comment (jonaharagon) [reply docs/android.md:178]: Xiaomi and Oppo do 4 years and Vivo does 3 on their flagship models. Budget/mid-range phones from various manufacturers might still fall in the 2 year camp. If we want to compare to those that's fine:

```suggestion
    Beginning with the **Pixel 8** and **8 Pro**, Pixel devices receive a minimum of 7 years of guaranteed security updates, ensuring a much longer lifespan compared to the 2-5 years competing OEMs typically offer.
```

Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply docs/android.md:178]: Only reason i mentioned that is that 8a will probably also have 7 years of updates and that falls under budget devices

**Review by jonaharagon — APPROVED** [review]

**Review by dngray — APPROVED** [review]

**Pull request merged**